packages/core/components/FileList/index.tsx

```diff
@@ -152,15 +153,27 @@ export default function FileList(props: FileListProps) {
     }, [fileSet]);
 
     const fileFetchWrapper = async (startIndex: number, endIndex: number) => {
+        setLocalError(undefined); // reset
         try {
             await fileSet.fetchFileRange(startIndex, endIndex);
         } catch (err) {
-            props.dispatch(setError(err as Error, true));
+            props.dispatch(setError(err as Error, isRoot));
+            // Root has its own error handling
+            if (!isRoot) setLocalError(err as Error);
             throw err;
         }
     };
 
     let content: React.ReactNode;
+    if (!!localError) {
+        return (
+            <div className={classNames(styles.container, className)}>
+                <div className={styles.errorMessage}>
+                    Sorry, some files could not load. Error: {localError.message}
```

Comment:
This message feels a little strange to me. I dont think that we need to say sorry. Something along the lines of "Some files could not be loaded." seems more appropriate to me.